src/types/types.ts

```diff
@@ -1093,6 +1093,17 @@ export interface EdgeSwapQuote {
   readonly request: EdgeSwapRequest
 
   readonly isEstimate: boolean
+
+  /**
+   * This quote may be partially fulfilled with remaining source funds left
+   * in wallet
+   */
+  readonly isMaybePartial?: boolean
```

Comment:
Would `canBePartial` or `allowsPartial` be better names? We use the word "maybe" a lot with cleaners, where it has a somewhat different connotation. Otherwise the name you have would be fine.

Reply:
`canBePartial` sounds good. will make the change.